views/ancestorLines/ancestor_tree.js

```diff
@@ -146,9 +150,11 @@ export class AncestorTree {
             console.log(
                 `Retrieving getPeople result page. keys:..., ancestors:${depth}, start:${start}, limit:${limit}`
             );
-            const [, ancestor_json] = await API.getPeople(reqIds, depth, start, limit);
+            starttime = performance.now();
```

Comment:
This is giving me an error in the console: 

Uncaught (in promise) TypeError: Assignment to constant variable.